mono/profiler/mono-profiler-log.c

```diff
@@ -1379,48 +1389,26 @@ process_requests (MonoProfiler *profiler)
 		mono_gc_collect (mono_gc_max_generation ());
 }
 
-// Avoid calling this directly. Use the functions below.
+// Avoid calling this directly if possible. Use the functions below.
 static void
-safe_send (MonoProfiler *profiler, gboolean if_needed, gboolean threadless)
+send_log_unsafe (MonoProfiler *profiler, gboolean lock, gboolean if_needed)
```

Comment:
More than one boolean per function is a code smell, It's better to use a single flags argument.